package.json

```diff
@@ -6,7 +6,7 @@
     "url": "https://github.com/UKHomeOffice/file-vault"
   },
   "engines": {
-    "node": ">=14.15.0"
+    "node": ">=18.12.1"
```

Comment:
Would it work with v22? Also if the docker image used is version 20, probably best we specify the same here

Reply:
@DivineTaminangHO - Please could you comment on this